docs/generator.md

```diff
@@ -39,7 +39,7 @@ minified               | boolean  | `false`         | Should the output be minif
 concise                | boolean  | `false`         | Set to `true` to reduce whitespace (but not as much as `opts.compact`)
 filename               | string   |                 | Used in warning messages
 jsonCompatibleStrings  | boolean  | `false`         | Set to true to run `jsesc` with "json": true to print "\u00A9" vs. "©";
-numbers                | string   | `decimal`       | Set `numbers: octal` in `jsescOption` to get the octal integer representation of any numeric value. Other supported options are `decimal, binary` and `hexadecimal`.
+jsescOption            | string   |                 | Given some data, jsesc returns an ASCII-safe stringified representation of that data. Refer https://github.com/mathiasbynens/jsesc#api for the shape of `jsesOption` and more details.
```

Comment:
```suggestion
jsescOption            | object   |                 | Use `jsesc` to process literals. Refer https://github.com/mathiasbynens/jsesc#api for the shape of `jsesOption` and more details.
```

Reply:
@JLHwung seems like your comment is out of date?

Also, your suggestion could be misread as "_if_ you pass an object, we'll use `jsesc` to process literals".

@sidntrivedi012 he's right re: string -> object though, we should make that change.

Reply:
@existentialism We use `jsesc` unconditionally on string literals if `jsescOption` presents, unlike numeric literals which is applied only when `jsesecOption.number` presents.

Reply:
> Use `jsesc` to process literals.

I was merely pointing out that the above statement could be read as if we only do it if you pass options, which isn't true?

Reply:
@existentialism @JLHwung Sorry for the delay in making changes. PTAL. Thanks :)

Reply:
@existentialism @JLHwung Sorry for the delay in making changes. PTAL. Thanks :)